### PR TITLE
Memefixes 2

### LINKED ISSUE #299: doctrine_earth_mother_mercy is shown during faith creation
**Your mod version is:**
c6c9e9ac

**What expansions do you have installed?**
None

**Are you using any submods/mods? If so, which?**
Graphical only

**Please explain your issue in as much detail as possible:**
doctrine_earth_mother_mercy group is shown during faith creation, despite the fact that there is only one doctrine that can't be changed or removed
Imo, special_doctrine_earth_mother_mercy either should go away when creating a new Faith, or moved to "special" group, in order to hide it on faith creation screen, just like how "naked priests" is not shown in vanilla in faith creator

**Steps to reproduce the issue:**
Try to reform Tauren religion

**Upload an attachment below: .zip of your save, or screenshots:**

<details>
<summary>Click to expand</summary>

image

</details>

### PR DESCRIPTION
**Pull request by MemeWizard**
## Changelog:
- Vrykuls now slay jalgars instead of trolls in northern lords flavor pack events
- Changed how unions tab looks, check new version and say what you think about it
- Because of how paradox handle articles in the game localisation (they don't write them), we had "End Justifies Means" as special doctrine name, which is uniformly correct with the game, while being grammatically incorrect. So it was renamed to "By Any Means Necessary".
- Fixes #299. Special doctrines are located on the top now.
  - Elunic "Gender stratification" and "Earth Mother's Mercy" are now special doctrines instead of the main doctrines
  - Added icon to "By Any Means Necessary" doctrine

## Developer changelog:
- Commented unused vanilla effects and modifiers
- Renamed shendralar_religion.xcf after it's author instead of me
- changed `has_trait = beauty_good` to `has_trait_beauty_good_trigger` in northern lords flavor pack files

## Tests:
- [x] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [x] The mod takes less than 5.5 GB in the Task Manager (Windows)

# How to test:

### CODE REVIEW

File: localization/replace/english/wc_dlc_fp1_l_english.yml [hunk 1]
```diff
@@ -57,4 +57,6 @@
  SteleCommemoratesAncestors_Gluttonous:0 "$SteleCommemoratesAncestors_Intro$. Not once in [second.MakeScope.Var('stele_container').Char.Custom('GetCustomAge')] did [second.MakeScope.Var('stele_container').Char.GetSheHe] skip a meal. $SteleCommemoratesCarver$."
  
  fp1_reprieve_from_sa_modifier:0 "Reprieve from Adventurers"
- fp1_reprieve_from_sa_modifier_desc:0 "Somehow, this character has won a measure of respite from the relentless raiding hordes.\n$EFFECT_LIST_BULLET$ Will not be attacked by landless Adventurers"
\ No newline at end of file
+ fp1_reprieve_from_sa_modifier_desc:0 "Somehow, this character has won a measure of respite from the relentless raiding hordes.\n$EFFECT_LIST_BULLET$ Will not be attacked by landless Adventurers" SteleCommemoratesAncestors_Gluttonous:0 "$SteleCommemoratesAncestors_Intro$. Not once in [second.MakeScope.Var('stele_container').Char.Custom('GetCustomAge')] did [second.MakeScope.Var('stele_container').Char.GetSheHe] skip a meal. $SteleCommemoratesCarver$."
```
Comment (zumbak04) [inline localization/replace/english/wc_dlc_fp1_l_english.yml:60]:
```suggestion
 fp1_reprieve_from_sa_modifier_desc:0 "Somehow, this character has won a measure of respite from the relentless raiding hordes.\n$EFFECT_LIST_BULLET$ Will not be attacked by landless Adventurers" 
SteleCommemoratesAncestors_Gluttonous:0 "$SteleCommemoratesAncestors_Intro$. Not once in [second.MakeScope.Var('stele_container').Char.Custom('GetCustomAge')] did [second.MakeScope.Var('stele_container').Char.GetSheHe] skip a meal. $SteleCommemoratesCarver$."
```

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: localization/english/gui/wc_gui_l_english.yml [hunk 2]
```diff
@@ -5,39 +5,39 @@
...
  UN_CUSTOM_LEADER_TITLE:0 "$UN_CUSTOM_LEADER_CULTURE$ [GetPlayer.MakeScope.Var('gui_custom_leader').GetCharacter.GetTitleAsName] practicing $UN_CUSTOM_LEADER_RELIGION$"
  UN_CUSTOM_LEADER_GODS:0 "[GetPlayer.MakeScope.Var('gui_custom_leader').GetCharacter.GetFaith.PantheonTerm]"
- 
+
  CV_CUSTOM_UNION:0 "[GetPlayer.MakeScope.Var('gui_custom_leader').GetCharacter.Custom( 'GetCustomUnionName' )]"
  CUSTOM_UNION_DESC:0 "$WCPLF_100_DESC_custom$"
- 
- UN_UNION_FEATURES:0 "#E #bold Features#!"
+
+ UN_UNION_FEATURES:0 "#Bold Features:#!"
```
Comment (zumbak04) [inline localization/english/gui/wc_gui_l_english.yml:36]:
```suggestion
 UN_UNION_FEATURES:1 "#Bold Features:#!"
```

File: gui/hud.gui [hunk 3]
```diff
@@ -225,240 +225,258 @@ widget = {
...
-						# DESC
-						vbox = {
-							layoutpolicy_horizontal = expanding
-							margin = { 10 5 }
-							spacing = 10
+								
+								# DESC
+								vbox = {
+									layoutpolicy_horizontal = expanding
+									margin = { 10 5 }
```
Comment (zumbak04) [inline gui/hud.gui:312]: To center the union title.
```suggestion
```
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

**Review by zumbak04 — APPROVED** [review]
LGTM! Good job!

**Review by ercarp — CHANGES_REQUESTED** [review]
> Vrykuls now slay jalgars instead of trolls in northern lords flavor pack events

Event `fp1_yearly.0081` still refers to _trolls_ in its localization. Should this be also changed to say _[jalgar](https://wowpedia.fandom.com/wiki/Jalgar)_?

<details><summary>Click to expand</summary>

image
image

</details>

Everything else looks fine to me. Big fan of the new Unions UI, everything is spaced out a bit better now.

**Review by ercarp — APPROVED** [review]
I noticed you added a few lines I had missed as well. I did a quick scour through Vanilla localization files and I could not find any more mentions of trolls that would need to be changed.

LGTM!

**Review by arithon — APPROVED** [review]
loc LGTM

File: localization/replace/english/wc_dlc_fp1_l_english.yml [hunk 4]
```diff
@@ -57,4 +57,11 @@
  SteleCommemoratesAncestors_Gluttonous:0 "$SteleCommemoratesAncestors_Intro$. Not once in [second.MakeScope.Var('stele_container').Char.Custom('GetCustomAge')] did [second.MakeScope.Var('stele_container').Char.GetSheHe] skip a meal. $SteleCommemoratesCarver$."
  
  fp1_reprieve_from_sa_modifier:0 "Reprieve from Adventurers"
- fp1_reprieve_from_sa_modifier_desc:0 "Somehow, this character has won a measure of respite from the relentless raiding hordes.\n$EFFECT_LIST_BULLET$ Will not be attacked by landless Adventurers"
\ No newline at end of file
+ fp1_reprieve_from_sa_modifier_desc:0 "Somehow, this character has won a measure of respite from the relentless raiding hordes.\n$EFFECT_LIST_BULLET$ Will not be attacked by landless Adventurers"
+ SteleCommemoratesAncestors_Gluttonous:0 "$SteleCommemoratesAncestors_Intro$. Not once in [second.MakeScope.Var('stele_container').Char.Custom('GetCustomAge')] did [second.MakeScope.Var('stele_container').Char.GetSheHe] skip a meal. $SteleCommemoratesCarver$."
```
Comment (hamjustine) [inline localization/replace/english/wc_dlc_fp1_l_english.yml:61]: this line already exist at 57

File: localization/replace/german/wc_dlc_fp1_l_german.yml [hunk 5]
```diff
@@ -57,4 +57,11 @@
  SteleCommemoratesAncestors_Gluttonous:0 "$SteleCommemoratesAncestors_Intro$. Niemals in  [second.MakeScope.Var('stele_container').Char.Custom('GetCustomAge')] hat [second.MakeScope.Var('stele_container').Char.GetSheHe] eine Mahlzeit ausgelassen. $SteleCommemoratesCarver$."
 
  fp1_reprieve_from_sa_modifier:0 "Befreit von Abenteurern"
- fp1_reprieve_from_sa_modifier_desc:0 "Irgendwie hat dieser Charakter eine gewisse Atempause von den unerbittlichen Plündererhorden gewonnen.\n$EFFECT_LIST_BULLET$ Wird nicht von landlosen Abenteurern angegriffen"
\ No newline at end of file
+ fp1_reprieve_from_sa_modifier_desc:0 "Irgendwie hat dieser Charakter eine gewisse Atempause von den unerbittlichen Plündererhorden gewonnen.\n$EFFECT_LIST_BULLET$ Wird nicht von landlosen Abenteurern angegriffen"
+ SteleCommemoratesAncestors_Gluttonous:0 "$SteleCommemoratesAncestors_Intro$. Not once in [second.MakeScope.Var('stele_container').Char.Custom('GetCustomAge')] did [second.MakeScope.Var('stele_container').Char.GetSheHe] skip a meal. $SteleCommemoratesCarver$."
```
Comment (hamjustine) [inline localization/replace/german/wc_dlc_fp1_l_german.yml:61]: this line also exist in double

File: localization/replace/english/wc_dlc_fp1_l_english.yml [hunk 6]
```diff
@@ -17,18 +17,18 @@
...
 
- fp1_yearly.0031.desc.hunted_troll:1 ""... 'course, this most recent wound is from the dire troll itself. Big as a house, quiet as a mouse, claws like the bastard spawn of [lying_warrior.GetFaith.DevilName]. Still, stab anything long enough, and it dies.""
- fp1_yearly.0071.desc.outro.loveli_lakes:0 "\n\nI have picked over the mess for only a short time when [loveli_lakes.GetTitledFirstName] steps forward.\n\n"This is no troll, [ROOT.Char.GetFirstNameNoTooltip], this is the work of a feral moose! I know the beast well, including its nasty bite, and would carve my mark on its corpse.""
- fp1_yearly.0071.desc.outro.fallback:0 "\n\nThe locals insist this is nothing less than the work of a dire troll, and demand I scour nearby woods to dispatch for the creature."
- fp1_yearly.0081.hunt_invalidated:0 "Hunt for the Dire Troll of [ROOT.Char.GetCapitalLocation.GetCounty.GetTitle.GetNameNoTierNoTooltip] Ceases"
+ fp1_yearly.0031.desc.hunted_troll:1 ""... 'course, this most recent wound is from the jalgar itself. Big as a house, quiet as a mouse, claws like the bastard spawn of [lying_warrior.GetFaith.DevilName]. Still, stab anything long enough, and it dies.""
```
Comment (hamjustine) [inline localization/replace/english/wc_dlc_fp1_l_english.yml:28]: Is it normal that there is double " around the sentence ?

Comment (arithon) [reply localization/replace/english/wc_dlc_fp1_l_english.yml:28]: I think that's meant as verbal speech there.

Comment (MemeWizard) [reply localization/replace/english/wc_dlc_fp1_l_english.yml:28]: Yes, it's the same in vanilla
AFAIK it's a part of event text with direct speech

File: localization/replace/english/wc_dlc_fp1_l_english.yml (hunk 4 above)
Comment (MemeWizard) [reply localization/replace/english/wc_dlc_fp1_l_english.yml:61]: Thanks for noticing! <3

**Comment by MemeWizard** [comment]
@hamjustine @arithon @ercarp @zumbak04 thanks for reviews <3

**Pull request merged**